pkgs/data/icons/gruvbox-plus-icon-pack/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenvNoCC, fetchFromGitHub, gtk3, breeze-icons, gnome-icon-theme, hicolor-icon-theme }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "gruvbox-plus-icon-pack";
+  version = "4.0";
+
+  src = fetchFromGitHub {
+    owner = "SylEleuth";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

```suggestion
    rev = "v${version}";
```
This works fine as is, just a style choice. It is more common not to use `refs/tags`

---

This actually was causing the build to fail for me

---

wasn't working bc of a typo, went ahead and updated

---

This was because of a typo in the package previously, so removing should work!